src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    add(a, b) {
+      return a + b;
+    },
+    subtract(a, b) {
+      return a - b;
+    },
+    multiply(a, b) {
+      return a * b;
+    },
+    divide(a, b) {
+      if (b === 0) {
+        return 'Cannot divide by zero';
+      }
+
+      return a / b;
```

Comment:
[CODE KNOWLEDGE] - The divide method should only take one parameter and divide the result property by this parameter using the /= operator. Additionally, it's good to check for division by zero, but the method should not return a string as an error message; it should throw an Error instead.